go-controller/pkg/metrics/master.go

```diff
@@ -234,33 +239,35 @@ func RegisterMasterMetrics(nbClient, sbClient goovn.Client) {
 	})
 }
 
-// StartE2ETimeStampMetricUpdater adds a goroutine that updates a "timestamp" value in the
-// nbdb every 30 seconds. This is so we can determine freshness of the database
+// StartE2ETimeStampMetricUpdater adds a goroutine that updates the nb_cfg "timestamp" value in the
+// nbdb every 30 seconds. This is so we can determine freshness of the database as well as
+// ovn-controller's status.
+// northd will propagate this down in to the sbdb. ovn-controller will update this value
+// in the sbdb when it synchronizes.
 func StartE2ETimeStampMetricUpdater(stopChan <-chan struct{}, ovnNBClient goovn.Client) {
 	startE2ETimeStampUpdaterOnce.Do(func() {
 		go func() {
+			klog.Infof("Starting periodic nb_cfg updater...")
 			tsUpdateTicker := time.NewTicker(30 * time.Second)
+			defer tsUpdateTicker.Stop()
+
+			tick := func() {
+				t := time.Now().Unix()
+				_, _, err := util.RunOVNNbctl("set", "NB_Global", ".", "nb_cfg="+strconv.FormatInt(t, 10))
+				if err != nil {
+					klog.Errorf("Failed to bump NB_Global nb_cfg value: %v", err)
+				} else {
+					klog.Infof("Set nb_cfg to %d", t)
```

Comment:
This will fill up log every 30s. We should remove this.

Reply:
I set it to V(7).